src/v/cluster/cluster_utils.h

```diff
@@ -273,6 +273,29 @@ inline bool has_non_replicable_op_type(const topic_table_delta& d) {
     }
     __builtin_unreachable();
 }
+
+inline std::vector<model::broker_shard> union_replica_sets(
```

Comment:
this is actually needed by the previous commit

Reply:
I thought no one would notice it in a backport :-P